### PR TITLE
Plot_selection_and_layout_V2

### PR DESCRIPTION
**Pull request by arunherb**
Updated the CVs for Plot Selection and Layout Module to align with V2 of the vocab changes

### CODE REVIEW

File: vocab_files/attribute_concepts/current-project-code.ttl
Comment (junrongYu) [inline vocab_files/attribute_concepts/current-project-code.ttl:1]: Hi @arunherb , Guru yesterday said that we can use only one attribute `project code`, what do you think? also, it's not recorded in the data endpoint https://core.vocabs.paratoo.tern.org.au/documentation#/Plot-selection & https://core.vocabs.paratoo.tern.org.au/documentation#/Plot-layout

**Comment by arunherb** [comment]
@junrongYu  - I agree. We should use `project code` for the `project(s) for plot` (in the Monitor App), irrespective of whether they are for current or past projects. And we can use `project codes` for `relevant project(s)` as a collection of all projects.

**Comment by junrongYu** [comment]
> @junrongYu - I agree. We should use `project code` for the `project(s) for plot` (in the Monitor App), irrespective of whether they are for current or past projects. And we can use `project codes` for `relevant project(s)` as a collection of all projects.

Thanks @arunherb , is the `project codes` a categorical collection or attribute?

**Comment by arunherb** [comment]
> > @junrongYu - I agree. We should use `project code` for the `project(s) for plot` (in the Monitor App), irrespective of whether they are for current or past projects. And we can use `project codes` for `relevant project(s)` as a collection of all projects.
> 
> Thanks @arunherb , is the `project codes` a categorical collection or attribute?

Hi @junrongYu - It is a categorical collection of all the relevant projects.

**Comment by junrongYu** [comment]
@junrongYu   will fix errors in methods while doing the review.

**Comment by junrongYu** [comment]
Hi @arunherb , are there any new LUTs in this module? Are there any values changed in exisiting LUTs?

File: vocab_files/attribute_concepts/fauna-plot-points.ttl
[the remark this answers is not in the document]
Comment (arunherb) [reply vocab_files/attribute_concepts/fauna-plot-points.ttl:1]: @junrongYu - this one is for 'fauna plot'. The `fauna plot points` is a different plot from the Core 1-ha plot (`plot points`) - based on the protocol.

[the remark this answers is not in the document]
Comment (junrongYu) [reply vocab_files/attribute_concepts/fauna-plot-points.ttl:1]: Hi @arunherb , I see you changed the prefLabel of this one to `plot points` too, does that mean they are the same? if so, only one concept file is needed

File: vocab_files/attribute_concepts/plot-marker-location.ttl
[the remark this answers is not in the document]
Comment (junrongYu) [reply vocab_files/attribute_concepts/plot-marker-location.ttl:1]: @junrongYu  will remove the attribute and the uri occurrence.

**Pull request merged**